roles/atomic_scan_verify/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
This test is kind of tricky because I believe that OpenSCAP can only scan RHEL containers and eventually I could see a future where RHEL containers are available only via a subscription.

I guess we can run with this for now.  I wonder if there is any benefit to using the Fedora OpenSCAP image in the Fedora/CentOS case?

https://hub.docker.com/r/fedora/atomic_scan_openscap/